### PR TITLE
Upgrade to Actix Web 4; support of env config interpolation

### LINKED ISSUE #117: Unexpected stops encountered every day
Error details

`
thread 'actix-rt:worker:1' panicked at 'called `Result::unwrap()` on an `Err` value: InvalidUriParts(InvalidUri(SchemeMissing))', /cargo/registry/src/github.com-1ecc6299db9ec823/actix-web-3.3.3/src/middleware/normalize.rs:161:46
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
`

### PR DESCRIPTION
**Pull request by aegenet**
Hello, this PR fixes an issue and adds a feature (_I can split these into two PRs if you prefer_)

- Upgrade of Vigil dependencies, particularly Actix Web from 3 to 4:

This will fix issue #117, which I encounter every day.

Resolves #117

- Support for environment variables with string interpolation in the configuration file.

For example: `manager_token: ${VIGIL_MANAGER_TOKEN}`.

### CODE REVIEW

**Comment by valeriansaliou** [comment]
Thank you so much! Will issue a release soon.

**Comment by williamdes** [comment]
Thank you so much !
This is a very important feature for private URLs in public repo config instead of having to do it manually: https://github.com/datacenters-network/status/blob/3005c9b0eb447df2ab8d1af1d2587618444086d2/docker-entrypoint.sh#L7

File: README.md [hunk 1]
```diff
@@ -145,6 +145,8 @@ Vigil will be reachable from `http://localhost:8080`.
 
 Use the sample [config.cfg](https://github.com/valeriansaliou/vigil/blob/master/config.cfg) configuration file and adjust it to your own environment.
 
+Note: you can use environment variables with string interpolation in your configuration file. For example: `manager_token: ${VIGIL_MANAGER_TOKEN}`.
```
Comment (williamdes) [inline README.md:148]: is the example syntax wrong ?
Should it not be `=` ?

Comment (valeriansaliou) [reply README.md:148]: I just tested interpolation and the documented syntax definitely works as expected.

Comment (williamdes) [reply README.md:148]: Yes but the toml syntax is not
```toml
manager_token: ${VIGIL_MANAGER_TOKEN}
```
But 
```toml
manager token = ${VIGIL_MANAGER_TOKEN}
```
Right

Comment (valeriansaliou) [reply README.md:148]: Ah, good catch! Fixed in 78100cae99a461a11a65444e918403533491f698

Comment (williamdes) [reply README.md:148]: Thank you!
One good example to show would be
`"icmp://${MY_SECRET_ADDR}"`

**Pull request merged**